Deck_Shuffler/Deck_Shuffler/Card.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Deck_Shuffler
+{
+    public class Card
+    {
+        public string value { get; }
```

Comment:
Convention is Pascal case for properties. So Value here.

Reply:
Because it's public right?